### ISSUE TITLE
Replace the default site .html file with some Symbiosis documentation

### ISSUE THREAD

**Issue by patch0** [BytemarkHosting/symbiosis#63]
@jamielinux's good idea;

Replace:

![screen_shot_2017-05-31_at_16 33 05](https://user-images.githubusercontent.com/317667/27084986-c99e5000-5045-11e7-9256-bd9f52cab638.png)

With help documentation in some form. Maybe full, maybe single page

Would a high level view of the steps required to setup a standard website work?

> Originally reported on [Bytemark's Gitlab](https://gitlab.bytemark.co.uk/open-source/symbiosis/issues/76) by @steveu on 2017-05-31T15:46:37.274Z

**Comment by patch0** [comment]
Yay!

So when this thought originally hit, I was thinking about mostly these two with the filezilla screenshots, and also possibly how to setup a DB:

* https://www.bytemark.co.uk/docs/symbiosis/connecting/
* https://www.bytemark.co.uk/docs/symbiosis/website-setup/
* https://www.bytemark.co.uk/docs/symbiosis/database-setup/

All on one page, made much briefer so that's it's not so cumbersome, and people can click through to bytemark.co.uk/docs if they need slightly more in-depth instructions. (Could even include the hostname in the instructions, because that can be detected/done on install.) And signposting to other stuff etc.

So basically, a brief, easy to follow quickstart guide to getting a simple site up within minutes. Could possibly even use setting up a basic/empty WordPress site as the ongoing example (ie, install filezilla, upload wordpress site, create the appropriate database via phpmyadmin or something like that).

> Originally posted by @jamielinux on 2017-05-31T16:14:41.073Z

**Comment by ianeiloart** [comment]
Why not just a page confirming you server is set up, and then an iframe that loads a page that we can fix when we find bad information.

**Comment by telyn** [comment]
@ianeiloart that's probably a good idea - seems like the complaints about iframes are purely SEO-based which doesn't matter for this. It could be part of the documentation repo then

**Comment by patch0** [comment]
I think this could be solved by #35.  Incidentally #79 fixed the "missing default page for SSL hosted domains".
